lib/utils.js

```diff
@@ -168,3 +169,18 @@ exports.checkForUpdates = () => {
   }
   return null;
 };
+
+
+/**
+ * Gets random IPv6 Address from a block
+ *
+ * @param {string} block
+ * @returns {string}
+ */
+exports.getRandomIPv6 = block => {
+  let subnet = parseInt(block.split('/')[1]);
+  block = block.split('/')[0];
+  if (!subnet || subnet > 128 || subnet < 24) throw Error('Invalid IPv6 subnet');
+  // Other errors will be thrown by ip6 itself
+  return ip6.randomSubnet(block, subnet, 128, 1, true)[0];
```

Comment:
i see we're not first rolling the /64 net
like it

Reply:
yea it was a requested change by you so i changed 👀